test/DynamoCoreTests/SchedulerTests.cs

```diff
@@ -935,6 +935,7 @@ public void TestTaskStateChangedEventHandling()
     public class SchedulerIntegrationTests : UnitTestBase
     {
         private DynamoModel dynamoModel;
+        private Preloader preloader;
```

Comment:
Unit tests now rely on `DynamoShapeManager.Preloader` for its geometry preloading activities.